<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>站酷小薇LOGO体</family>
    <pointsize>36</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{background-color: #e6f3f3;}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>站酷小薇LOGO体</family>
      <pointsize>48</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SETTING</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>站酷小薇LOGO体</family>
      <pointsize>36</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>修改姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>站酷小薇LOGO体</family>
      <pointsize>36</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>文字速度</string>
    </property>
   </widget>
   <widget class="QLabel" name="sound_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>站酷小薇LOGO体</family>
      <pointsize>36</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>游戏音量</string>
    </property>
   </widget>
   <widget class="QLabel" name="teach_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>站酷小薇LOGO体</family>
      <pointsize>36</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>游戏教程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>660</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#save_button{
	border-image: url(:/image/resource/save_button2.png);}
#save_button:hover{
	border-image: url(:/image/resource/save_hover.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>660</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#exit_button{
	border-image: url(:/image/resource/exit_button2.png);}
#exit_button:hover{
	border-image: url(:/image/resource/exit_hover.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="view_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#view_button{background:qlineargradient(spread:pad,x1:0,y1:1,x2:0,y2:0,stop:0 #e4f1f3,stop:1#bcf0c4);
border-radius:12px;}
#view_button:hover{
background:#bcf0c4;
border-radius:12px;}</string>
    </property>
    <property name="text">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QPushButton" name="high_speed_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#high_speed_button{background:qlineargradient(spread:pad,x1:0,y1:1,x2:0,y2:0,stop:0 #e4f1f3,stop:1#bcf0c4);
border-radius:12px;}
#high_speed_button:hover{
background:#bcf0c4;
border-radius:12px;}</string>
    </property>
    <property name="text">
     <string>快速</string>
    </property>
   </widget>
   <widget class="QPushButton" name="middle_speed_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#middle_speed_button{background:qlineargradient(spread:pad,x1:0,y1:1,x2:0,y2:0,stop:0 #e4f1f3,stop:1#bcf0c4);
border-radius:12px;}
#middle_speed_button:hover{
background:#bcf0c4;
border-radius:12px;}</string>
    </property>
    <property name="text">
     <string>中速</string>
    </property>
   </widget>
   <widget class="QPushButton" name="low_speed_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>280</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#low_speed_button{background:qlineargradient(spread:pad,x1:0,y1:1,x2:0,y2:0,stop:0 #e4f1f3,stop:1#bcf0c4);
border-radius:12px;}
#low_speed_button:hover{
background:#bcf0c4;
border-radius:12px;}</string>
    </property>
    <property name="text">
     <string>慢速</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>551</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>400</y>
      <width>50</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource2.qrc"/>
 </resources>
 <connections/>
</ui>
